frontend/src/components/AddEventForm.js

```diff
@@ -0,0 +1,96 @@
+import {Autocomplete, Button, TextField} from "@mui/material";
+import {DatePicker, LocalizationProvider} from "@mui/lab";
+import DateAdapter from "@mui/lab/AdapterMoment";
+import styled from "styled-components";
+import {useEffect, useState} from "react";
+import axios from "axios";
+import * as events from "events";
+
+export default function AddEventForm() {
+
+    const cities = [
+        {label: 'Hamburg'},
+        {label: 'Berlin'},
+        {label: 'Stuttgart'},
+    ];
+
+    const genres = [
+        {label: 'Concerts'},
+        {label: 'Sport'},
+        {label: 'Comedy'},
+    ];
+
+    const [eventToAdd, setEventToAdd] = useState({
+        name: "",
+        city: "",
+        genre: "",
+        date: "",
+    })
+
+    const handleNameChange = (event) => {
+        setEventToAdd({...eventToAdd, name: event.target.value})
+    }
+
+    const handleCityChance = (event, value) => {
+        setEventToAdd({...eventToAdd, city: value})
+    }
+
+    const handleGenreChange = (event, value) => {
+        setEventToAdd({...eventToAdd, genre: value})
+    }
+
+    const handleDateChange = (event) => {
+        setEventToAdd({...eventToAdd, date: event._d.toJSON()})
+    }
```

Comment:
you could simplify those methods to a single, more generic method. 

```suggestion
    const handleChange = (event) => {
        setEventToAdd({...eventToAdd, [event.target.name] : event.target.value})
    }
```

In order to make this function work you need to add the `name` attribute to your input fields.